src/pgo/parser/TLAParser.java

```diff
@@ -1613,46 +1582,76 @@ private static PGoTLAUnit getLastUnit(Mutator<LocatedList<PGoTLAUnit>>... mutato
 		}
 		return lastUnit;
 	}
+
+	static final Pattern TLA_BEGIN_TRANSLATION = Pattern.compile("\\\\\\*+\\s+BEGIN TRANSLATION$", Pattern.MULTILINE);
+	static final Pattern TLA_END_TRANSLATION = Pattern.compile("\\\\\\*+\\s+END TRANSLATION$", Pattern.MULTILINE);
```

Comment:
I agree with the trailing whitespace part, but I would remind you that since those backslashes are in a string literal they need to be escaped. If you "adjust" the string to be unescaped, it looks like this: \\\*+\s+BEGIN TRANSLATION$

or, the string "\" followed by one or more "*" (rest should be obvious).

Reply:
(will fix the whitespace soon, still fixing some bug I caused)